shared/common-adapters/icon.constants.js

```diff
@@ -750,6 +750,11 @@ const iconMeta_ = {
     extension: 'gif',
     require: require('../images/icons/icon-loader-uploading-16.gif'),
   },
+  'icon-loader-uploading-static-16': {
+    isFont: false,
+    extension: 'gif',
+    require: require('../images/icons/icon-loader-uploading-static-16.gif'),
```

Comment:
I'm confused by this. Isn't there an `'icon-loader-uploading-16-static'` above? And why are the extensions of this one ".gif"?